### ISSUE TITLE
differerent domain

### ISSUE THREAD

**Issue by ilpepe** [ncjones/python-guerrillamail#2]
hi,
it would be nice to change the domain, as on the guerilla website.

could you point were i should start, as i don't have an idea right now...

**Comment by ncjones** [comment]
Do you mean you want to use a different base URL for the Guerillamail API? You can change this with the "base_url" property in the `~/.guerrillamail` config file.

**Comment by ilpepe** [comment]
blazing fast response! thx.

well the thing is guerillamailblock.com doenst seem to work, sharklasers does for my 'application'.

so i would like to use sharklases.com as a domain name...

i'm unable to find the .guerrillamail file. this supposed to be created when installing the package?
i'm using ubuntu 14.04

**Comment by ncjones** [comment]
The ~/.guerrillamail file is only used by the CLI, not by the Python client. I've updated the readme to describe this. It is created automatically after the first use of the CLI.

AFAIK, the sharklasers.com and guerrillamailblock.com domains are funnelled into the same guerrillamail server and that guerillamail is not using the domain name to identify recipients. For example, if I set my address to "nathan@sharklasers.com" then I can receive messages sent to either "nathan@sharklasers.com" or "nathan@guerrillamailblock.com" (or any other of the guerillamailblock.com aliases).

Does this help?
